docs/docs/setting-up/networking-instructions/fundamentals.md

```diff
@@ -11,31 +11,44 @@ Bacalhau uses [libp2p](https://libp2p.io/) under the hood to communicate with ot
 
 Because bacalhau is built using libp2p, the concept of peer identity is used to identify nodes on the network.
 
-When you start a bacalhau node using `bacalhau serve`, it will look for an RSA private key in the `~/.bacalhau` directory. If it doesn't find one, it will generate a new one and save it there.
+Bacalhau uses 2 keys to identify the node and the user:
+- A private Ed25519 `libp2p_private_key` key is used to get the Peer ID in libp2p
+- A private RSA key in the PKCS1 format `user_id.pem` is used to uniquely identify user on the network when making reqeusts from CLI
 
-You can override the directory where the private key is stored using the `BACALHAU_PATH` environment variable.
+By default, Bacalhau creates its own keys on installation. These keys can be found at the path given by the config:
+- key `User.KeyPath` for a RSA key, which by default is `~/.bacalhau/user_id.pem`
+- key `User.Libp2pkeypath` for an Ed25519 key, which by default is `~/.bacalhau/libp2p_private_key`
 
-Private keys are named after the port used for the libp2p connection which defaults to `1235`. By default when first starting a node, the private key will be stored in `~/.bacalhau/private_key.1235`.
+
+When you start a bacalhau node using `bacalhau serve`, it will look for these keys in the paths, specified in the environmental variable or config. If it doesn't find them, it will generate a new ones and save them there.
```

Comment:
the server command doesn't need the user_id.pem, and so really it's just looking for the libp2p_private_key, creating it if necessary.

Reply:
fixed